modules/calcite/README.txt

```diff
@@ -4,7 +4,36 @@ Apache Ignite Calcite Module
 Apache Ignite Calcite module provides experimental Apache Calcite based query engine.
 
 To enable Calcite based engine explicit `CalciteQueryEngineConfiguration` instance should be added to
-`SqlConfiguration.QueryEnginesConfiguration` property (see `SqlConfiguration.setQueryEnginesConfiguration()`).
+`SqlConfiguration.QueryEnginesConfiguration` property (see `SqlConfiguration.setQueryEnginesConfiguration()`) or ./config/queries-default.xml.
+Config example:
+
+<bean id="queries.cfg" class="org.apache.ignite.configuration.IgniteConfiguration">
+    <property name="clientConnectorConfiguration">
+        <bean class="org.apache.ignite.configuration.ClientConnectorConfiguration">
+            <property name="host" value="127.0.0.1"/>
+            <property name="port" value="10800"/>
+            <property name="portRange" value="10"/>
+        </bean>
+    </property>
```

Comment:
This part looks redundant for SQL configuration documentation

Reply:
explicit always preferable and more readable that implicit i just want to show dependency between port in config and jdbc url.